docs/faq.mdx

```diff
@@ -24,6 +24,14 @@ Head over to [our installation page](/installation-getting-started/what-am-i-ins
 
 ---
 
+## How many resources does Pieces use on your machine?
+
+Due to our commitment to providing a very secure experience by leveraging on-device models, Pieces does require a decent amount of resources; 2GB is fairly standard for what we offer today. We are always working to make Pieces more performant, but this is just due to the nature of our implementation.
+
+To make things a little better, you can select "Optimize Memory Usage" in the PiecesOS menu by clicking on the PiecesOS icon in your taskbar. This setting can help reduce the memory footprint and improve performance on your machine.
```

Comment:
Pieces OS

Reply:
I've updated "PiecesOS" to "Pieces OS" for consistency. Thanks! 😊